moped-editor/src/queries/project.js

```diff
@@ -102,6 +102,19 @@ export const SUMMARY_QUERY = gql`
   }
 `;
 
+export const STATUS_QUERY = gql `
+  query StatusQuery {
```

Comment:
i've been meaning to ask y'all, do we ever make use of [hasura caching](https://hasura.io/docs/latest/graphql/cloud/response-caching.html) for our lookup value queries? cc @mateoclarke @frankhereford @sergiogcx @tillyw

Reply:
I didn't know it existed until now, it seems to be a feature of hasura cloud only? @johnclary 

>Query response caching is available for Hasura Cloud projects on the Standard (pay-as-you-go) tier and above

Reply:
We don't afaik. I believe its a Hasura Cloud (not Core) feature and only v2+.

Reply:
ah, my bad! thanks

Reply:
on second read, it seems like it is configurable in the open source engine, but not sure what setup is required.
https://hasura.io/learn/graphql/hasura-advanced/performance/1-caching/

Reply:
i see, hasura makes use of postgres's caching abilities. however caching at the api layer is only a cloud feature.

regardless, apollo's cache should at least be caching these queries during the user's session, right?

Reply:
Yes, apollo caches the majority of requests locally (in the browser of the user). If we wanted server-side cache, we can put it behind CloudFront and use http headers for controlling cache hit/miss and ttl.